src/analysis/seminorm.lean

```diff
@@ -708,6 +714,60 @@ section normed_field
 variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E] (p : seminorm 𝕜 E) {A B : set E}
   {a : 𝕜} {r : ℝ} {x : E}
 
+lemma bdd_below_range_add (x : E) (p q : seminorm 𝕜 E) :
+  bdd_below (range (λ (u : E), p u + q (x - u))) :=
+by { use 0, rintro _ ⟨x, rfl⟩, exact add_nonneg (p.nonneg _) (q.nonneg _) }
+
```

Comment:
this should not be in the part on seminorm balls. You probably need to add a new section after line 590.

Reply:
It shouldn't exist at all.

Reply:
I was just highlighting the beginning of the whole section that is in the wrong place.